### DISCUSSION TITLE
BoF meetup at CppCon 2024 - Monday 16th @ 12:15

### DISCUSSION THREAD

**Discussion by bluetarpmedia** [hsutter/cppfront#1220] [Suggestions]
Last year we had an informal meetup to discuss cppfront/Cpp2 topics and it was well attended.

Is anyone going to CppCon this year? We could do something similar:
- Show off a pull request you're working on and get feedback
- Present on a topic you're passionate about
- Chat in person about some of the Cppfront GitHub bugs & discussion topics

One of the lunchtime Open Content sessions could work nicely.

**Comment by hsutter** [comment]
Good idea!

@bluetarpmedia could you drive submitting an Open Content session request?

Everyone: If you're attending CppCon and have something Cpp2/cppfront-related you might like to present, please reply here with your idea, and then we can divvy up the time slot (I didn't do as good a job of organizing last year, this time we can put together a rough agenda with times).

**Reply by bluetarpmedia** [reply]
Sure thing, do you have a preferred day?

**Reply by hsutter** [reply]
Good point: Earlier in the week is more reliable for me, so Mon-Wed would be ideal.

**Reply by bluetarpmedia** [reply]
Done, I'll post here when I hear back.

**Reply by gregmarr** [reply]
I will be there, don't necessarily have anything to present, but will be sure to be there for the discussion.

**Comment by bluetarpmedia** [comment]
CppCon have scheduled the Cpp2 BoF meetup for Monday 16th at 12:15.

**Comment by bluetarpmedia** [comment]
I'd be happy to present 2 metafunctions I've been working on.

**Comment by DyXel** [comment]
Hey all! how did it go? Anything cool showcased or spoken?

**Reply by bluetarpmedia** [reply]
It was a smaller turn out compared to last year, maybe 10 of us, but we had some good discussions. I presented 2 metafunctions for struct-of-arrays and external polymorphism, then there were various questions about Cpp2 which Herb answered, and finally Herb finished by talking about his plans to extend metafunctions so they can be applied to things other than just types.
